config/schema_descriptions.yml

```diff
@@ -0,0 +1,72 @@
+reporting_patient_states:
+  description: "Monthly summary of a patient's information and health indicators. This table has one row per patient, per month, from the month of the patient's registration."
+  columns:
+    patient_id: "ID of the patient"
+    recorded_at: "Time (in UTC) at which the patient was registered"
+    status: "active, dead, migrated, etc"
+    gender: "Gender of the patient"
+    age: "Age of the patient as entered by a nurse"
+    age_updated_at: "Time (in UTC) at which the age was last updated"
+    date_of_birth: "Date of birth of the patient"
+    current_age: "Computed age as of today, given age, age-updated-at and date-of-birth. This will have the same value for a given patient across all rows."
+    month_date: "Date at the beginning of the month for which this row has the latest information of a patient"
+    month: "Month (1-12) of year"
+    quarter: "Quarter (1-4) of year"
+    year: "Year in YYYY format"
+    month_string: "String that represents a month, in YYYY-MM format"
+    quarter_string: "String that represents a month, in YYYY-Q format"
+    hypertension: "Yes if patient is hypertensive, no if not, and nil if the data is unavailable"
```

Comment:
Consider a more conceptual description:

> Has the patient been diagnosed with hypertension (yes/no/unknown). Nil if the data is unavailable.

Also, let's make sure to use the correct enum values with the right case as well.

Reply:
Obv same for all medical history questions

Reply:
Yeah, made this change :) Would be nice to have enums for the field so that we don't have to explain the data type in the description. Perhaps in a later update.